src/components/AsideHeader/components/PageLayout/PageLayout.tsx

```
@@ -26,6 +46,13 @@ const Layout = ({compact, className, children, topAlert}: PageLayoutProps) => {
                     ...({'--gn-aside-header-size': `${size}px`} as React.CSSProperties),
                 }}
             >
+                {typeof preloadHeightValue === 'number' ? (
+                    <style
+                        dangerouslySetInnerHTML={{
+                            __html: `.g-root{--gn-top-alert-height:${preloadHeightValue}px;}`,
+                        }}
+                    />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)